spec/install/gemspecs_spec.rb

```diff
@@ -46,4 +46,76 @@
     bundle :install, :artifice => "endpoint_marshal_fail" # force gemspec load
     should_be_installed "activesupport 2.3.2"
   end
+
+  context "when ruby version is specified in gemspec and gemfile" do
+    it "installs when patch level is not specified and the version matches" do
+      build_lib("foo", :path => tmp.join("foo")) do |s|
+        s.write "Gemfile", <<-G
+        source "file://#{gem_repo1}"
+        ruby '#{RUBY_VERSION}', :engine_version => '#{RUBY_VERSION}', :engine => 'ruby'
+        gemspec
+        G
+        s.required_ruby_version = RUBY_VERSION
+      end
+
+      Dir.chdir(tmp.join("foo")) do
+        bundle "install"
+        should_be_installed "foo 1.0"
+      end
+    end
+
+    it "installs when patch level is specified and the version still matches the current version" do
+      build_lib("foo", :path => tmp.join("foo")) do |s|
+        s.write "Gemfile", <<-G
```

Comment:
I looked for a way to set `Gem::Specification#required_ruby_version` with `install_gemfile`, but I could not find one. Is it possible?